R/fit.R

```diff
@@ -707,21 +710,23 @@ CmdStanFit$set("public", name = "constrain_variables", value = constrain_variabl
 #' evaluated at a posterior draw (which is on the constrained space). `lp__` is
 #' intended to diagnose sampling efficiency and evaluate approximations.
 #'
-#' `lp_approx__` is the log density of the variational approximation to `lp__`
-#' (also on the unconstrained space). It is exposed in the variational method
-#' for performing the checks described in Yao et al. (2018) and implemented in
-#' the \pkg{loo} package.
+#' For variational inference `lp_approx__` is the log density of the variational
+#' approximation to `lp__` (also on the unconstrained space). It is exposed in
+#' the variational method for performing the checks described in Yao et al.
+#' (2018) and implemented in the \pkg{loo} package.
+#'
+#' For Laplace approximation `lp_approx__` is the the unnormalized density of
+#' the Laplace approximation.
```

Comment:
Add: It can be used to perform the same checks as in the case of the variational method described in Yao et al. (2018).

Reply:
Thanks, just pushed a commit adding this